hs_core/page_processors.py

```diff
@@ -34,6 +35,16 @@ def get_page_context(page, user, resource_edit=False, extended_metadata_layout=N
 
     metadata_status = _get_metadata_status(content_model)
 
+    relevant_tools = []
+    for tool in ToolResource.objects.all():
+        for res_type in tool.metadata.res_types.all():
```

Comment:
I see what you mean now. Thanks for explaining that again. Does the change I just committed (https://github.com/hydroshare/hydroshare/commit/f53aa0870b8c5c278a2ce920c67e28edd09c49be) address that?